internal/metadata/k8sfiles.go

```diff
@@ -38,6 +40,9 @@ func (i K8sFilesLoader) UpdatePlan(inputPath string, plan *plantypes.Plan) error
 
 	files, err := common.GetFilesByExt(inputPath, []string{".yml", ".yaml"})
 	if err != nil {
+		if errors.Is(err, syscall.EACCES) { // if err is permission denied error then return it
```

Comment:
I think it's best to always return here. @ashokponkumar what do you think? previously we were just warning and continuing https://github.com/konveyor/move2kube/blob/master/internal/metadata/k8sfiles.go#L41

Reply:
Or at least add `IsNotExist` to the list of errors which cause an early return
https://golang.org/pkg/os/#IsNotExist

Other than that it LGTM

Reply:
Yes, since GetFilesByExt returns an error only if there is an error in the base input path, the fileslist will anyway be empty. So we can return early in all cases.

Reply:
@vovapi can you change it just return early if there is an error? No need to check if it `EACCESS`

Reply:
@vovapi Can you possibly remove this check and update the PR?